src/main/kotlin/com/github/imflog/schema/registry/RegistryClientWrapper.kt

```diff
@@ -15,14 +15,16 @@ object RegistryClientWrapper {
 
     private var registryClient: SchemaRegistryClient? = null
 
-    fun client(url: String, userInfo: String): SchemaRegistryClient? {
+    private const val basicAuthSourceUserInfo: String = "USER_INFO"
```

Comment:
const in Kotlin should be Uppercased. So BASIC_AUTH_SOURCE should be good.
Also I would prefer if that const was located close to the SchemaRegistryBasicAuth class.